tools/README.md

```diff
@@ -84,7 +84,16 @@ One of the tasks concatenates all the .js files in `/srcjs` together into `/inst
 
 During development of Shiny's Javascript code, it's best to use `grunt watch` so that the minified file will get updated whenever you make changes the Javascript sources.
 
+#### Auto build and browser refresh
 
+An alternative to `grunt watch` is to use `entr` to trigger `grunt` when sources
```

Comment:
In our projects, the .md files generally don't use hard line breaks within a paragraph.

Also, this only works on Mac, right? It would be good to mention that here.